PyTado/interface/api/my_tado.py

```diff
@@ -224,6 +225,46 @@ def get_zone_overlay_default(self, zone: int):
 
         return self._http.request(request)
 
+    def set_home(self) -> None:
```

Comment:
afaik the presence stuff is universal between v3/v2 systems and Tado X systems so it should stay in the abstract base class so it can be used by both the `Tado` and the `TadoX` class